e2e/Makefile

```diff
@@ -92,6 +92,12 @@ kube-prometheus:
 
 .PHONY: clean
 clean: stop-lvmd
+	for d in $$($(SUDO) find $(TMPDIR) -type f); do \
```

Comment:
Shouldn't it be "-type d"?

Reply:
"type -f" is correct. There are only file mount points left because the directory mount points are already unmounted here (called by `make stop-lvmd` -> `make -C $(TMPDIR)/topolvm/example stop-lvmd` -> `make -C $(TMPDIR)/topolvm/example shutdown-kind`)
https://github.com/topolvm/topolvm/blob/e00d2f66180b9058e3c78268ce095de34805291f/example/Makefile#L76-L80

The variable name "d" may be changed to "f".

Reply:
OK, I see.

Reply:
I've changed variable name `d` to `f`.